components/MaskedTransition/examples/MaskedTransitionTypicalUse.swift

```diff
@@ -117,26 +119,27 @@ open class MaskedTransitionTypicalUseSwiftExample: UIViewController {
 
   var transitionController: MDCMaskedTransitionController? = nil
   func didTapFab(fab: UIView) {
-    let target = targets[tableView.indexPathForSelectedRow!.row]
+    guard let indexPathForSelectedRow = tableView.indexPathForSelectedRow else { return }
+    let target = targets[indexPathForSelectedRow.row]
     let vc = target.viewControllerType.init()
-
+    
```

Comment:
thanks for committing the improvement! Just a few small things, there are now tabs here in the empty lines that should be reverted.